packages/edit-post/src/components/header/tools-more-menu-group/index.js

```diff
@@ -15,7 +10,7 @@ const { Fill: ToolsMoreMenuGroup, Slot } =
 ToolsMoreMenuGroup.Slot = ( { fillProps } ) => (
 	<Slot fillProps={ fillProps }>
 		{ ( fills ) =>
-			! isEmpty( fills ) && (
+			fills.length > 0 && (
```

Comment:
Nope, it will always be an array:

https://github.com/WordPress/gutenberg/blob/a5102012f9c9bef1488c13045914edf9d4d4fbcf/packages/components/src/slot-fill/slot.js#L70-L92